src/Features/VisualBasic/Portable/VBFeaturesResources.resx

```diff
@@ -1250,4 +1250,7 @@ Sub(&lt;parameterList&gt;) &lt;statement&gt;</value>
   <data name="If_statement_can_be_simplified" xml:space="preserve">
     <value>'If' statement can be simplified</value>
   </data>
+  <data name="Add_Shadows" xml:space="preserve">
+    <value>Add Shadows</value>
```

Comment:
nit.  it really feels like this should be: Add 'Shadows'

Reply:
Or `Add 'Shadows' Keyword`? Not sure if it's a lesser known keyword that might confuse people if they see the message. (Why do I want my code casting shadows on the ground???)

Reply:
➡️ Current form follows the existing `Add_Overloads` line. Can make this a separate design proposal.

Reply:
@jasonmalinowski we've been moving to a simpler form of expressing these concepts in lightbulbs.  For example ```Invert 'if'``` instead of ```Invert 'if' statement```.  And ```Convert to 'foreach'``` Instead of ```Convert 'for' statement to 'foreach' statement```.

In general, we think that a simple 'verb' and 'language-construct-name' is generally sufficient to get the point across clearly without being too verbose.

--

Ideally though, we'd just be consistent here across all our features.  That's hte highest order bit for me, regardless of what style we think is best.

Reply:
OK, happy to count this as is if that's the intent. But :+5000000: to fixing Add Overloads while we're here.

Reply:
With your :+5000000: and my :+5000000: we're up to :+10000000:.  And pretty soon, we're talking about real fictional internet points...

(pretty plz, can we just do this :) )

Reply:
Looks awesome. Agree `Add 'Overloads'`

Reply:
Fixed, and the commit where I fixed it reveals bigger localization problems than just the quotes...